src/Tizen.NUI/src/public/BaseComponents/ViewInternal.cs

```diff
@@ -1076,6 +1076,23 @@ internal virtual void ApplyBorderline()
         {
             if (backgroundExtraData == null) return;
 
+            // ActionUpdateProperty works well only if BACKGROUND visual setup before.
+            // If view don't have background visual, we set transparent background color in default.
+            using(PropertyMap tmp = new PropertyMap())
+            {
+                using(var propertyValue = Object.GetProperty(SwigCPtr, Property.BACKGROUND))
+                {
+                    if(!propertyValue.Get(tmp) || tmp.Empty())
```

Comment:
When View.BackgroundColor is not set, tmp is empty.
Therefore, in that case, !propertyValue.Get(tmp) is not necessary.

Reply:
Thank you. I found some more general solution of this problem - BackgroundProperty.
It already did same thing what my code did. So I replace the codes, and remain only `tmp.Emtpy()` now.